src/funcx_common/response_errors/error_classes.py

```diff
@@ -356,3 +356,36 @@ class InsufficientAuthScope(FuncxResponseError):
     def __init__(self) -> None:
         self.error_args = []
         self.reason = "Insufficient scope for this action"
+
+
+class BatchRunTooLarge(FuncxResponseError):
+    """Raised when the size of the POST body sent to the batch_run API
+    exceeds our limits."""
+
+    code = ResponseErrorCode.BATCH_RUN_TOO_LARGE
+    http_status_code = HTTPStatusCode.PAYLOAD_TOO_LARGE
+
+    def __init__(self, payload_size: int, max_size: int):
+        self.error_args = [payload_size, max_size]
+        self.reason = f"batch submission is too large ({payload_size}/{max_size} bytes)"
```

Comment:
I think the other reason texts are all capitalized. ie. f"Batch submission..."

Reply:
I find `/` slightly ambiguous (even if context and/or trial-and-error would resolve it).  Consider adding a space and using a greater-than character (&gt;):
```python
self.reason = f"... too large ({payload_size} B > {max_size} B)"
```